tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -564,8 +567,21 @@ private String handleUnicodeSet(XPathParts parts, String xpath, String value) {
                 examples.add(LRM + "➖ " + SUSF.format(winning_minus_value));
             }
         }
+        if (parts.containsAttributeValue("type", "auxiliary")) {
+            LanguageTagParser ltp = new LanguageTagParser();
+            String ltpScript = ltp.set(cldrFile.getLocaleID()).getResolvedScript();
+            UnicodeSet exemplars = ScriptToExemplars.getExemplars(ltpScript);
+            UnicodeSet main = cldrFile.getExemplarSet(ExemplarType.main, WinningChoice.WINNING);
+            UnicodeSet mainAndAux = new UnicodeSet(main).addAll(valueSet);
+            if (!mainAndAux.containsAll(exemplars)) {
+                examples.add(
+                        LRM
+                                + "🔬️ "
```

Comment:
What's this ?

Reply:
It is a microscope. I was hunting around for some symbol that could be used to distinguish that line, and give a hint that it was a suggestion for addition. I used an emoji, like in other cases, to save space in the display. Glad to have other alternatives.

Reply:
a `title="…` attribute explaining it might be helpful also.